Assets/Scripts/OnClickEvents/ClickToToggleLight.cs

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+
+
+public class ClickToToggleLight : MonoBehaviour
+{
+    public UnityEngine.Rendering.Universal.Light2D light2D;  // Assign the 2D light GameObject in the Inspector
```

Comment:
use import instead lol

Reply:
this does not work. It bugs out the whole screen because the import does not exist.